public/app/features/alerting/unified/Analytics.ts

```diff
@@ -219,6 +219,25 @@ export function trackRulesSearchComponentInteraction(filter: keyof RulesFilter)
 export function trackRulesListViewChange(payload: { view: string }) {
   reportInteraction('grafana_alerting_rules_list_mode', { ...payload });
 }
+export const trackSwitchToSimplifiedRouting = async (
+  props: AlertRuleTrackingProps = {
```

Comment:
I'm not sure if we need that. It looks like these metadata is attached by default to any interaction report

Reply:
yep, I decided to keep it in case we need at some point, but I think you're right